README.md

```diff
@@ -216,4 +197,4 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Your MIT link is still not fixed. Kindly do that.

  ![image](https://user-images.githubusercontent.com/54848363/220669313-30b65853-a01a-4108-923e-39f63bc084ab.png)

Reply:
sorted